src/Config/Validator/Deploy/DeprecatedVariables.php

```diff
@@ -60,7 +73,10 @@ public function validate(): Validator\ResultInterface
             );
         }
 
-        if (isset($variables[DeployInterface::VAR_SCD_EXCLUDE_THEMES])) {
+        // default value for SCD_EXCLUDE_THEMES is an empty string
+        if (isset($config[DeployInterface::VAR_SCD_EXCLUDE_THEMES])
+            && $config[DeployInterface::VAR_SCD_EXCLUDE_THEMES] !== ''
```

Comment:
If you want to add notification only when variable has some value then you also need to include empty array.
But I would notify customer in any case if this variable exists with any value. The same as for VAR_STATIC_CONTENT_SYMLINK

Reply:
Again though, `MergedConfig` includes this variable's default value if none is set by the user. Testing if it's not `''` allows us to check that the variable has been modified.